package.json

```diff
@@ -48,6 +48,7 @@
     }
   },
   "devDependencies": {
+    "@nitric/sdk": "^0.1.22",
```

Comment:
Is there a way we could use import re-wiring for the snippets instead to achieve this? i.e. setup an alias in the snippets to point to lib or src for tests? Seems a bit weird for the lib to depend on itself for development.

Reply:
I have replaced with tsconfig paths